Gopkg.lock

```diff
@@ -0,0 +1,610 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
```

Comment:
Does this file need to be on git if it's autogenerated?

Reply:
gopkg.lock specifies the version of every package in the vendor dir. The default behaviour is download the latest version that may change from build to build. I think it's probably better to have it in the git repo and keep the control on the vendor versions.

Reply:
Ok, we can leave it in this PR, and then get rid of it when migating to GO modules that has an specific file for deps (and allow to have code outside of $GOPATH). Take a look in this post: https://blog.liquidbytes.net/2018/09/quick-and-easy-guide-for-migrating-to-go-1-11-modules/